idc_index/index.py

```diff
@@ -6,13 +6,17 @@
 import subprocess
 import tarfile
 import zipfile
+import duckdb
 
 
 class IDCClient:
     def __init__(self):
         current_dir = os.path.dirname(os.path.abspath(__file__))
         file_path = os.path.join(current_dir, 'idc_index.csv.zip')
-        self.index = pd.read_csv(file_path, dtype=str, encoding='utf-8')
+        if not os.path.exists(file_path):
```

Comment:
@vkt1414 I added this and s5cmd detection below to facilitate local testing.

Reply:
I do check whether csv and s5cmd installed properly in the setup.py..
![image](https://github.com/ImagingDataCommons/idc-index/assets/115020590/2eae7894-898d-4f75-91c6-6021831e72b1)
![image](https://github.com/ImagingDataCommons/idc-index/assets/115020590/a940070b-36ad-4d76-84a6-2893e9ce4005)